CKB-Explorer-PRDs/Glossary/Ongoing.md

```diff
@@ -0,0 +1,54 @@
+| 区块详情页 | Cellbase for Block | The cellbase transaction of block N is sent to the miner of block N-11 as reward, learn more from our [Consensus Protocol](https://docs.nervos.org/docs/basics/concepts/consensus/) | checked(sven) | Current hint |
+| 交易详情页 | Status | Status represent the status of current transaction, They are Pending, Comfirming and Rejected | checked(sven) | - |
+| 交易详情页 | CellDeps | CellDeps is one of dependency in the transaction data structure, it allows scripts in the transaction to access (read-only) referenced live cells. For more info, check about [Code Locating](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0022-transaction-structure/0022-transaction-structure.md#code-locating) | checked(sven) | [Dependencies](https://docs.nervos.org/docs/essays/dependencies/#how-dependencies-work) |
+| 交易详情页 |OutPoint| The OutPoint is a reference type. The transaction uses OutPoint in inputs to reference the previously created cells instead of embedding them. |checked(sven)|[transaction struc](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0022-transaction-structure/0022-transaction-structure.md#code-locating)|
+| 交易详情页 | OutPoint.Index | OutPoint Index is the one of the params which are used to locate a Cell  | checked(sven) | - |
+| 交易详情页 | DepType | DepType in CKB is a field in CellDep to differentiate the normal cells which provide the code directly, and the dep groups which is expanded to its members inside cell_deps. If the uint8 value is 0, the cell provide the code, if the uint8 value is 1, the cell includes a dep group | checked(sven) | [transaction struc](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0022-transaction-structure/0022-transaction-structure.md) |
+| 交易详情页 | dep_group | dep_group is a method for referencing multiple dependencies which are commonly used together using a single dependency field. It is used to simplify the process of adding multiple dependencies to a transaction. | To be confirmed | [transaction struc](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0022-transaction-structure/0022-transaction-structure.md) |
+| 交易详情页 | OutPoint.TxHash | OutPoint.TxHash is a field that represents the transaction hash of a particular output Cell in a transaction. It's used to locate a cell with OutPoint.Index | checked(sven) | - |
+| 交易详情页 | HeaderDeps | header_deps is one of dependency in the transaction data structure, it allow scripts in the transaction to access (read-only) data of referenced past block headers of the blockchain.| checked(sven) | [Dependencies](https://docs.nervos.org/docs/essays/dependencies/#how-dependencies-work) |
+| 交易详情页 | Witness | Witness is a field in the CKB transaction structure that contains the signature in a transaction| checked(sven) | - |
```

Comment:
Not a accurate statement here, witness often contains a signature, not always. And we can add a ref link here 
https://github.com/nervosnetwork/ckb/blob/3de5a20ce60619927f41f81d9584cab9d39d1275/util/types/schemas/blockchain.mol#L114-L118